### ISSUE TITLE
please update an important information about file restore instance on Cloud Backup Service

### ISSUE THREAD

**Issue by tnaokiJP** [NetAppDocs/occm#641]
Page: [Learn about Cloud Backup](https://docs.netapp.com/us-en/occm/concept_backup_to_cloud.html)
note : detail point is here "Home>Documentation>Cloud Docs>Cloud Manager>Learn about Cloud Backup>Cost - Resource charges>for file restore"

I know that restore instance use the instance type which decided by Public Cloud, but some end-users want to know what OS and build version is used too. They must concern about the cost of cloud service, OS and build version affect it.

note : AWS – Amazon Linux 2, Azure – Cent OS, Google Cloud - ???
please add some information of build version and update this page.

regards,

**Comment by netapp-tonacki** [comment]
Hi tnaokiJP,  Thanks for your comment.  We don't support file restore for GCP yet, but I'll find out the OS and version for deployed instances in AWS and Azure.  I'll get back to you when I have the answers.

Glad to see that you joined to add your first comment.

Thanks, Tom

**Comment by netapp-tonacki** [comment]
Hello again tnaokiJP,  I have updated the topic with the OS/version for the Restore instance.  LIke you said, AWS is Amazon Linux 2.  Azure is CentOS 7.5.  GCP wil be added when we have single file restore support.

Based on your question, we have verified the documented OS for the Cloud Manager Connector instances and are getting the information for OS/version for the Cloud Data Sense instances.  So thanks for bring up this important addition to the docs.

Thanks, Tom
